zrml/prediction-markets/src/lib.rs

```diff
@@ -717,6 +735,14 @@ mod pallet {
 
         type ApprovalOrigin: EnsureOrigin<<Self as frame_system::Config>::Origin>;
 
+        /// The base amount of currency that must be bonded in order to create a dispute.
+        type DisputeBond: Get<BalanceOf<Self>>;
+
+        /// The additional amount of currency that must be bonded when creating a subsequent
+        /// dispute.
+        type DisputeFactor: Get<BalanceOf<Self>>;
```

Comment:
Moving back these parameters from `simple-disputes` to `PM` because they are both used by `simple-disputes` and `court` "back-ends". This feature will avoid duplication across different pallets